components/BucketListDefault.tsx

```diff
@@ -0,0 +1,18 @@
+import Image from 'next/image';
+import styles from '../styles/pageStyles/bucketList.module.css'
```

Comment:
[sand] We should start doing like `'@/styles/.......` so that we don't have to change imports even if we move this file